src/App.js

```diff
@@ -1,25 +1,55 @@
-import logo from './logo.svg';
+import React, { Component } from 'react';
 import './App.css';
+// import Header from './Header.js';
+import axios from 'axios';
+import Form from 'react-bootstrap/Form'
+import Button from 'react-bootstrap/Button'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+require('dotenv').config();
+
+class App extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      queryCity: '',
+      locationObject: {},
+      error: false
+    }
+  }
 
+  getLocation = async() => {
+    try {
+      let result = await axios.get(`https://us1.locationiq.com/v1/search.php?key=${process.env.password}&q=${this.state.queryCity}&format=json`);
```

Comment:
You had the right instincts on your key 💯 Double check that you have stored your key properly inside your `.env` file. See this link here for keys [React API KEYS how to](https://create-react-app.dev/docs/adding-custom-environment-variables/).